README.md

```diff
@@ -30,14 +30,17 @@ the steps accordingly.
     ```
 
 3.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
-    [Rocket](https://rocket.rs/), which means it needs a nightly version of Rust:
+    [Rocket](https://rocket.rs/), which means it needs a version of Rust:
```

Comment:
This should still say "a nightly version", it doesn't say which, after all. What it says now doesn't make much sense, of course it needs *a* version.

Reply:
reworded and rebased the branch